python/ray/rllib/models/catalog.py

```diff
@@ -63,17 +63,21 @@ def get_action_dist(action_space, dist_type=None):
             dist_dim (int): The size of the input vector to the distribution.
         """
 
+        # TODO(ekl) are list spaces valid?
+        if isinstance(action_space, list):
```

Comment:
@eugenevinitsky should the action spaces for multi-agent envs be lists or Tuples?

Reply:
The action spaces are lists, the observation spaces are tuples. The observation space was to take advantage of the tuple flattening preprocessor. This isn't very consistent so I'll submit a pull request fixing this.

Reply:
Ok cool, feel free to remove this conversion later if you think just supporting Tuple is sufficient.